examples/rest/v2/pulse.rb

```diff
@@ -0,0 +1,47 @@
+require_relative '../../../lib/bitfinex.rb'
+
+client = Bitfinex::RESTv2.new({
+  :url => ENV['REST_URL'],
+  :api_key => ENV['API_KEY'],
+  :api_secret => ENV['API_SECRET']
+})
+
+# Get pulse profile
+p client.get_pulse_profile('Bitfinex')
+
+# Get public pulse history
+p client.get_public_pulse_history({ :limit => 5 })
+
+# Submit new Pulse message
+pulse = client.submit_pulse({
```

Comment:
this is a "bigger issue" not directly related to this PR.

when the api key way used with the node lib, the nonce does not work any more:

https://github.com/bitfinexcom/bfx-api-node-util/blob/master/lib/nonce.js#L11

was added in https://github.com/bitfinexcom/bfx-api-node-util/commit/8980fe2e02cf471703f18acb75ed369c54efd2eb and is imho the wrong approach fixing it, because multiplying by 1000 does not fix the issue.


the ruby lib still has the old code, where a normal timestamp is used

Reply:
opened https://github.com/bitfinexcom/bitfinex-api-node/issues/566